.env.test

```
@@ -11,3 +11,5 @@ ENABLE_DEFAULT_OTP=true
 API_SECRET_TOKEN=sample-api-token
 NEXTAUTH_SECRET=sample-nextauth-secret
 NEXTAUTH_URL=https://test.flexile.dev:3101
+NEXT_PUBLIC_DOMAIN=flexile.dev
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_DOMAIN key should go before the QUICKBOOKS_WEBHOOK_SECRET key

(UnorderedKey)